cmd/compose/compose.go

```diff
@@ -315,7 +315,7 @@ func RootCommand(streams command.Cli, backend api.Service) *cobra.Command { //no
 			switch ansi {
 			case "never":
 				progress.Mode = progress.ModePlain
-			case "tty":
+			case "always":
```

Comment:
Shouldn't we support both `tty` and `always`?
```suggestion
			case "tty", "always":
```

Reply:
we could, but as this value never has been documented, and `SetANSIMode` doesn't manage this value, I considered we better should not pretend this even existed